test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -735,6 +744,140 @@ func testDeploymentRollback(c *testContext) {
 	}
 }
 
+// testPodConnectSubresource verifies connect subresources
+func testPodConnectSubresource(c *testContext) {
+	podGVR := gvr("", "v1", "pods")
+	pod, err := createOrGetResource(c.client, podGVR, c.resources[podGVR])
+	if err != nil {
+		c.t.Error(err)
+		return
+	}
+
+	// check all upgradeable verbs
+	for _, verb := range []string{"GET", "POST"} {
```

Comment:
would it be less confusing to use `method` instead of `verb` here? The rest of the code considers `verb` as admission's operations.

Reply:
renamed